src/AppBundle/Model/Event.php

```diff
@@ -616,6 +616,27 @@ public function getWikis(): Collection
         return $this->wikis;
     }
 
+    /**
+     * Get all of this event's wikis that do not yet have at least one category defined.
+     * Wikidata is excluded because it can never have categories.
+     * @return Collection Collection of EventWiki objects.
+     */
+    public function getWikisWithoutCategories(): ArrayCollection
```

Comment:
The return type declaration also needs to be `Collection`. I think if there are EventWikis that are persisted but haven't been flushed to the database, the `$this->wikis->filter` code will return a `PersistentCollection`, which in turn will cause an exception.

Reply:
Oops! Of course. :) Done.